package.json

```diff
@@ -56,7 +56,7 @@
   },
   "devDependencies": {
     "elm": "0.19.1-3",
-    "elm-format": "0.8.3",
+    "elm-format": "^0.8.4",
```

Comment:
should we really add the carret here?

Reply:
I don't know, if that would make some differences, worth noting.
Can you please tell, what this carret actualy does?

Reply:
Oh, I'm very sorry, I haven't seen the question before.
So, the carret signs that it will only do updates that do not change the leftmost non-zero number.
Example:
if you run `npm update` on `^0.13.0`, it would install versions `0.13.1`, `0.13.2`, etc, but never `0.14.0`
if you run `npm update` on `^1.13.0`, it would install `1.13.1`, `1.14.0`, etc, but never `2.0.0`

If you should use carrets or not will vary from project to project. Since it was not being used before, I would avoid adding it. :smile: